sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -66,6 +66,43 @@ export class ChangeFeedIterator<T> {
     get hasMoreResults(): boolean;
 }
 
+// @public
+export interface ChangeFeedIteratorOptions {
+    // Warning: (ae-forgotten-export) The symbol "ChangeFeedResource" needs to be exported by the entry point index.d.ts
+    changeFeedResource?: ChangeFeedResource;
+    // Warning: (ae-forgotten-export) The symbol "ChangeFeedStartType" needs to be exported by the entry point index.d.ts
+    changeFeedStartType?: ChangeFeedStartType;
+    maxItemCount?: number;
+    sessionToken?: string;
+}
+
+// @public
+export class ChangeFeedIteratorResponse<T> {
+    get activityId(): string;
+    get continuationToken(): string;
+    readonly count: number;
+    headers: CosmosHeaders;
+    get requestCharge(): number;
+    readonly result: T;
+    get sessionToken(): string;
+    readonly statusCode: number;
+    readonly SubStatusCode?: number;
```

Comment:
Why is the statusCode lower camel case but SubStatusCode upper cammel case? Is this some pattern particular to JS?

Reply:
Typo, fixed now.